articles/how-to-fix-and-prevent-sql-injection-in-wordpress/index.md

```diff
@@ -0,0 +1,196 @@
+### Introduction
+SQL injection in [WordPress](https://kinsta.com/knowledgebase/what-is-wordpress/) is [ranked the second most critical vunerability](secure.wphackedhelp.com). [Updating your WordPress website](https://wordpress.org/support/article/updating-wordpress/) to the latest release shields you from SQL injection. Nevertheless, third party themes and plugins still pose a big threat to WordPress users. When you hear SQL injection, you understand it is the manipulation of a security flaw in database systems. WordPress as a platform runs server-side scripts in PHP. SQl and PHP hence make WordPress vulnerable to attackers via inserting malicious URLs embedded with commands that trigger behaviours from the database. Consequently,  sensitive information from your database is exposed or leaked to hackers which they can use to manipulate your WordPress website.
+
+#### What is SQLi?
+
+SQLi is an abbreviation for SQL injection.
+
+SQL injection is the injection or insertion of SQL code usually via data transmitted from a website. When an SQL injection attack is deployed successfully, it gives them access to sensitive information as well as ability to add/edit/delete data from a database. Generally, SQLi gives an attacker the possibility to execute all actions related to a database. SQLi takes advantage of PHP and SQl database hence posing a big risk to WordPress. Other than PHP, [the ASP language](https://www.w3schools.com/asp/default.ASP) is also prone to SQLi.
```

Comment:
"SQLi takes advantage of PHP and SQl database hence posing a big risk to WordPress." is sort of a repetition of what you said in the first paragraph about WordPress being vulnerable due to using SQL and PHP.